src/Microsoft.ML.TorchSharp/NasBert/TextClassificationTrainer.cs

```diff
@@ -812,27 +842,83 @@ private sealed class Mapper : MapperBase
                         _inputColIndices.Add(col);
 
                 _labelCol = inputSchema[_parent._options.LabelColumnName];
+                _inputSchema = inputSchema;
+
+                torch.random.manual_seed(1);
+                torch.cuda.manual_seed(1);
```

Comment:
>cuda [](http://example.com/codeflow?start=22&length=4)

this seed need to be set only when we cuda? I assume this not expensive anyway but I guess it is a pinvoke call.

Reply:
If cuda is not enabled its a NOOP (at least thats my understanding). So it shouldn't be anymore expensive to do this.